tokens/contracts/erc-1155/erc-1155-meta/ERC1155RaribleMeta.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "../../../../exchange-v2/contracts/EIP712MetaTransaction.sol";
```

Comment:
we can't have dependencies between projects(folders, exchangev2 and tokens here). need to add EIP712MetaTransaction.sol to token folder or to "@rarible"

Reply:
agree. do it